webui/react/src/components/kit/NoteCard.tsx

```diff
@@ -6,31 +6,29 @@ import { unstable_useBlocker as useBlocker } from 'react-router-dom';
 import Button from 'components/kit/Button';
 import Input from 'components/kit/Input';
 import Tooltip from 'components/kit/Tooltip';
+import Markdown from 'components/Markdown';
 import Spinner from 'shared/components/Spinner/Spinner';
 import { ErrorType } from 'shared/utils/error';
 import handleError from 'utils/error';
 
-import Markdown from './Markdown';
-import css from './NotesCard.module.scss';
+import css from './NoteCard.module.scss';
 
-interface Props {
+export interface Props {
   disabled?: boolean;
   extra?: React.ReactNode;
   noteChangeSignal?: number;
   notes: string;
   onChange?: (editedNotes: string) => void;
   onSave?: (editedNotes: string) => Promise<void>;
   onSaveTitle?: (editedTitle: string) => Promise<void>;
-  style?: React.CSSProperties;
   title?: string;
 }
 
-const NotesCard: React.FC<Props> = ({
+const NoteCard: React.FC<Props> = ({
```

Comment:
Can we make `NoteCard` and `NoteCards` into one `Notes` component and use a parameter to indicate whether multiple notes are allowed?

Reply:
Maybe a better way of thinking about this is that we should have have a `Notes` component and a param that indicates whether adding new notes is allowed? Otherwise, you'd have to type the `notes` and `onSave` handlers to accept both singular and multiple notes based off of the param.

Reply:
Yes, that's fine. I was speaking from a UX standpoint rather than a code standpoint.

Reply:
You could do that though, if you wanted:
```typescript
type NoteProps = {
   multiple: true,
   onSave: (Array<Note>) => void,
} | {
   multiple: false,
   onSave: (Note) => void,
}
```

Reply:
I see, thanks